apps/Vision/Capture.py

```diff
@@ -1,4 +1,35 @@
-class Capture:
+import cv2
+import base64
+import sys
+import time
+import importlib
+
+if importlib.find_loader("picamera"):
+    found_picamera = True
+    from picamera import PiCamera
+    from picamera.array import PiRGBArray
+else:
+    found_picamera = False
 
+class Capture:
     def __init__(self):
+        self.camera_feed = 0
+        self.picamera = None
+        if found_picamera:
+            # initialize the camera and grab a reference to the raw camera capture
+            self.picamera = PiCamera()
+            self.picamera.resolution = (3280, 2464)
```

Comment:
Hey can we make the resolution a class property? And set the default to `(2592 , 1944)`? I would say make it a constructor value, but use the default if no value is provided

Reply:
Hey @avesk , is there a reason you want to keep the resolution lower than 8mp?